content/en/docs/contribute/participate/pr-wranglers.md

```diff
@@ -84,7 +84,7 @@ To close a pull request, leave a `/close` comment on the PR.
 
 {{< note >}}
 
-The [`fejta-bot`](https://github.com/fejta-bot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.
+The [`k8s-ci-robot`](https://github.com/k8s-triage-bot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.
```

Comment:
Yeah, it's @k8s-triage-robot instead of @k8s-ci-robot.
Your last commit make this inconsistent.
```suggestion
The [`k8s-triage-robot`](https://github.com/k8s-triage-robot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.
```

Reply:
Sorry, I'll fix it, this has been a rabbit hole when all I was looking to do was make a quick fix. I got a little turned around